src/hotspot/share/gc/g1/g1ConcurrentMark.cpp

```diff
@@ -1288,7 +1288,7 @@ class G1UpdateRegionLivenessAndSelectForRebuildTask : public WorkerTask {
           reclaim_empty_humongous_region(hr);
         }
       } else if (hr->is_old()) {
-        hr->note_end_of_marking(_cm->top_at_mark_start(hr), _cm->live_bytes(hr->hrm_index()));
+        hr->note_end_of_marking(_cm->top_at_mark_start(hr), _cm->live_bytes(hr->hrm_index()), _cm->incoming_refs(hr->hrm_index()));
```

Comment:
Potentially put `hr->hrm_index()` into a local so that the line is not that long any more; or just change the parameters to take a `HeapRegion*` for both - afaict they both are always called with `hr->hrm_index()`...